res/lang/gen_settings/genSettings_en.txt

```diff
@@ -83,15 +83,15 @@ REACH      = Reach
 PRICE      = Price
 VALUE      = Value
 POPULATION = Population
-NAME       = name
+NAME       = Name
 
 GENDER_MEN         = Men
 GENDER_WOMEN       = Women
 GENDER_ALL         = All
 GENDER_MEN_SHORT   = M
 GENDER_WOMEN_SHORT = W
 
-CURRENCY     = €
+CURRENCY     = $
```

Comment:
Yeah, I had thought that.  Currency doesn't seem well tied into localization.  That said I'd say that this would be a sensible change as long as it is tied in.  Almost no-one in the Anglosphere uses Euros.

Reply:
Well, they have to if they travel. IF we plan to move the currency to the map, for me it would be a not to change it now. But this is a personal opinion/feeling. @GWRon ?

Reply:
I think the "CURRENCY" is something we will sooner or later move out of the XML-Files anyways. I would support "similar" currencies (Dollar, Euro ... maybe GBP) in a hardcoded way. This won't need any "multiplier" then.

in Transport Tycoon (Deluxe) you were able to change the currency regardless of the "city names" on the map. But there values/prices "changed" accordingly. Somewhere in the forums, on discord or here in the issues we had discussed that already I think. Any simple "multiplier" (x1, x2, x3...) will not lead to "decimal numbers" and with "low multipliers" the same "length" of money values is more likely.

btw: https://github.com/TVTower/TVTower/pull/953
(left and right currency string).

so for now I would like the "currency" sign to not be touched - any maybe we could discuss it in a new issue and try out stuff like a `money_value=$%value%` approach (and somewhere also a "format" to use for thousands separators etc.)

Reply:
OK changed it back.